frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepageView.tsx

```diff
@@ -36,14 +35,8 @@ export type EmbedHomepageViewProps = {
 };
 
 export const EmbedHomepageView = (props: EmbedHomepageViewProps) => {
-  const {
-    embeddingAutoEnabled,
```

Comment:
we should also remove this setting in other places e.g.

- https://github.com/metabase/metabase/blob/eee00cff556e393680684cb1d3983d89882e6895/frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepage.tsx#L20

- https://github.com/metabase/metabase/blob/eee00cff556e393680684cb1d3983d89882e6895/enterprise/frontend/src/embedding-sdk/cli/steps/setup-embedding-settings.ts#L21
- https://github.com/metabase/metabase/blob/eee00cff556e393680684cb1d3983d89882e6895/frontend/src/metabase/setup/actions.ts#L247
- https://github.com/metabase/metabase/blob/eee00cff556e393680684cb1d3983d89882e6895/src/metabase/embed/settings.clj#L225


We could also create a follow up task to deal with this since I kind of feel this might involve more LoC change and test fixes.

Reply:
Yeah that's what I meant with the last sentence of the pr description:
> I'll create a follow up PR to change the setup to not send the embedding-autoenabled setting and to clean up related code and tests, this PR is to unblock master
But depending on if we care about what people on v50 see if they upgrade and are still seeing the embedding homepage, we may need to keep the actual setting.

I'm not sure but I think there may be needed some BE migration to properly delete a setting?

Reply:
> I'm not sure but I think there may be needed some BE migration to properly delete a setting?

I don't think we need to migrate as we're removing the setting, we technically only need to think about downgrading, but again, I think this is only a setting that's written once, so we probably don't need to worry about it too much. Consulting our BE engineer should give us more confident on the topic.

Reply:
I agree that we don't need to migrate this. Even downgrading will be fine. If we do feel the need to delete it, (though I can't see a good reason at this time), all we need to do is delete the defsetting's source code add its name to `metabase.models.setting/retired-setting-names`, so we don't accidentally re introduce a setting with the same name.